README.md

```diff
@@ -1,15 +1,15 @@
 # READ FIRST
-This project has reached EOL, which means that no new features will be developed. We're only merging bug fixes and/or security pull requests. You can read more about this decision by joining our [official Discord server](https://discord.gg/powercord).
+This project has reached EOL, which means that no new features will be developed. We're only merging bug fixes and/or security pull requests. You can read more about this decision by joining our [official Discord server](https://discord.gg/replugged).
```

Comment:
I wish :(

Reply:
This text has been fully removed so no longer an issue